src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -300,7 +303,7 @@ public Task<Document> AddEventAsync(Solution solution, INamedTypeSymbol destinat
             return GetEditAsync(
                 solution,
                 destination,
-                (t, opts, ai) => AddEvent(t, @event, opts, ai),
+                (t, opts, ai, _) => AddEvent(t, @event, opts, ai),
```

Comment:
So if the outer loop is already checking cancellation token, then why do only _some_ in the inner loop also check cancellation token? At first glance this seems arbitrary, since both are doing effectively the same work.